code/game/turfs/closed/walls.dm

```diff
@@ -150,6 +157,20 @@
 					"<span class='hear'>You hear a booming smash!</span>")
 	return TRUE
 
+/**
+  *Deals damage back to the hulk.
+  *
+  *When a hulk manages to break a wall using their hulk smash, this deals back damage to the arm used.
+  *This is in its own proc just to be easily overridden by other wall types. Default allows for three
+  *smashed walls per arm. Also, we use CANT_WOUND here because wounds are random. Wounds are applied
+  *by hulk code based on arm damage and checked after an attack completes.
+  *Arguments:
+  **arg1 is the arm to deal damage to.
+ */
+
```

Comment:
```suggestion
```
If there's a space between proc and documentation, it doesn't gets stored properly.